composer.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "danskernesdigitalebibliotek/openplatform",
+    "description": "Client for the Open Platform service for the danish libraries",
+    "type": "library",
```

Comment:
Should we add a [`bin` section](https://getcomposer.org/doc/articles/vendor-binaries.md) to the package definition to expose the `openplatform` command to consumers?

Reply:
It won't work without dev-dependencies. And it feels wrong to pull in silly and yaml for everybody.

Reply:
:+1: I see your point